### PR TITLE
DropShadow action support

### PR DESCRIPTION
**Pull request by magdakwiecien**
#### What does this PR solve?
- `e_dropshadow` action support

#### Final checklist
- [x] Implementation is aligned to Spec.
- [x] Tests - Add proper tests to the added code.

### CODE REVIEW

**Comment by magdakwiecien** [comment]
I don't know why it's failing 🤷 it passes locally.

**Review by const-cloudinary — APPROVED** [review]

**Comment by const-cloudinary** [comment]
The errors are pretty similar to these:
https://stackoverflow.com/questions/72717949/error-in-node-modules-types-babel-traverse-index-d-ts68-50-error-ts1005

File: src/actions/effect.ts [hunk 1]
```diff
@@ -454,6 +455,19 @@ function backgroundRemoval(): BackgroundRemoval {
 }
 
 
+/**
+ * @summary action
+ * @description Adds a shadow to the object on an image.
```
Comment (carlevison) [inline src/actions/effect.ts:460]:
```suggestion
 * @description Adds a shadow to the object in an image.
```

File: src/actions/effect/DropShadow.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import {Action} from "../../internal/Action.js";
+import {QualifierValue} from "../../internal/qualifier/QualifierValue.js";
+import {Qualifier} from "../../internal/qualifier/Qualifier.js";
+import {IDropShadowModel} from "../../internal/models/IEffectActionModel.js";
+import {IActionModel} from "../../internal/models/IActionModel.js";
+
+/**
+ * @description Adds a shadow to the object on an image.
```
Comment (carlevison) [inline src/actions/effect/DropShadow.ts:8]:
```suggestion
 * @description Adds a shadow to the object in an image.
```

File: src/actions/effect/DropShadow.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  protected _actionModel: IDropShadowModel = {};
+
+  constructor() {
+    super();
+    this._actionModel.actionType = 'dropshadow';
+  }
+
+  /**
+   * @description
+   * Shadow azimuth. (Range: 0 to 360, Server default: 100)
```
Comment (carlevison) [inline src/actions/effect/DropShadow.ts:26]:
```suggestion
   * The direction the light is coming from to cause the shadow effect. (Range: 0 to 360, Server default: 100)
```

File: src/actions/effect/DropShadow.ts [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+   * @return {this}
+   */
+  azimuth(azimuth?: number): this{
+    this._actionModel.azimuth = azimuth;
+    this._azimuth = azimuth;
+    return this;
+  }
+
+  /**
+   * Shadow elevation. (Range: 0 to 90, Server default: 70)
```
Comment (carlevison) [inline src/actions/effect/DropShadow.ts:37]:
```suggestion
   * The height of the light source above the 'ground' to cause the shadow effect. (Range: 0 to 90, Server default: 70)
```

File: src/actions/effect/DropShadow.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+   */
+  elevation(elevation?: number): this {
+    this._actionModel.elevation = elevation;
+    this._elevation = elevation;
+    return this;
+  }
+
+  /**
+   * @description
+   * Shadow spread. (Range: 0 to 100, Server default: 50)
```
Comment (carlevison) [inline src/actions/effect/DropShadow.ts:49]:
```suggestion
   * The spread of the light source. A small number means 'point' light. A larger number means 'area' light. (Range: 0 to 100, Server default: 50)
```

**Pull request merged**